extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -222,6 +222,7 @@ export class SafeframeHostApi {
             'ck_on': 1,
             'flash_ver': '26.0.0',
             'canonical_url': this.maybeGetCanonicalUrl(),
+            'is_amp': true,
```

Comment:
Ideally we would have an AMP object under which AMP specific items (e.g. canonical_url) would be placed.  For now how about you rename this to amp, place canonical_url within, and then once we the change is made within SF, you can just remove current canonical_url?

Reply:
done